src/main/java/edu/whimc/positiontracker/Tracker.java

```diff
@@ -55,11 +55,11 @@ public boolean startRunner() {
 		}
 
 		taskID = Bukkit.getScheduler().scheduleSyncRepeatingTask(this, () -> {
-			if (Bukkit.getOnlinePlayers().size() != 0) {
+			if (!Bukkit.getOnlinePlayers().isEmpty()) {
 				debugLog("Logging for " + Bukkit.getOnlinePlayers().size() + " player(s)...");
 				this.queryer.storePositionData();
 			}
-		}, 20 * 3, 20 * 3);
+		}, 20 * 2, 20 * 2); //every 2 seconds
```

Comment:
Ideally this value lives in the config so a new version doesn't have to be made every time we want to change this value.